envoy/buffer/buffer.h

```diff
@@ -507,21 +511,17 @@ class Instance {
 
 protected:
   /**
-   * Reserve contiguous memory segment with specific size. The first address of this memory segment
-   * will be returned.
+   * Reserve contiguous memory segment with specific size from the back slice. The first address of
+   * this memory segment will be returned. If there is no enough continuous memory in the back slice
+   * then this method will return nullptr. The reserved memory will be regarded as used and directly
+   * increase the length of the buffer.
+   *
    * @param expected contiguous memory segment size.
-   * @return the first address of this memory segment.
+   * @return the first address of this memory segment or nullptr if there is no enough continuous
```

Comment:
I think you can remove the part of the description that duplicates the return description.

Reply:
get it.